pkg/oonf_api/0007-Use-RIOT-s-container_of-implementation.patch

```diff
@@ -0,0 +1,45 @@
+From b2ad2073ac282f1bc6315e47ffbd12c3f6a9ae1a Mon Sep 17 00:00:00 2001
+From: Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+Date: Wed, 29 Oct 2014 11:37:05 +0100
+Subject: [PATCH] Use RIOT's container_of implementation
+
+---
+ src-api/common/container_of.h | 22 ++++++++++++++++++----
+ 1 file changed, 18 insertions(+), 4 deletions(-)
+
+diff --git a/src-api/common/container_of.h b/src-api/common/container_of.h
+index fcb38fe..b49d836 100644
+--- a/src-api/common/container_of.h
++++ b/src-api/common/container_of.h
+@@ -59,10 +59,24 @@
+  * @return pointer to surrounding struct
+  */
+ #ifndef container_of
+-#define container_of(ptr, type, member) ({ \
+-    const typeof(((type *)0)->member ) *__tempptr = (ptr); \
+-    (type *)((char *)__tempptr - offsetof(type,member)); \
+-  })
++#if __STDC_VERSION__ >= 201112L
```

Comment:
quick question: why is this the same as in *core/include/kernel_macros.h*. Is it possible to reduce the redundance anywhere?